packages/pages/src/build/build.ts

```diff
@@ -0,0 +1,26 @@
+import shell from "shelljs";
+import { CommandModule } from "yargs";
+import { ProjectFilepaths } from "../common/src/project/structure.js";
+
+type BuildArgs = Pick<ProjectFilepaths, "scope">;
+
+const handler = async ({ scope }: BuildArgs) => {
+  // Pass CLI arguments as env variables to use in vite-plugin
+  if (scope) {
+    process.env.YEXT_PAGES_SCOPE = scope;
+  }
+  shell.exec("vite build");
```

Comment:
does this require vite to be globally installed?

Reply:
no, `build` ran without vite globally installed on my machine